css/types/resolution.json

```diff
@@ -31,16 +45,18 @@
               "version_added": true
             },
             "safari": {
-              "version_added": true
+              "version_added": false,
+              "notes": "See <a href='https://webkit.org/b/78087'>bug 78087</a>."
             },
             "safari_ios": {
-              "version_added": true
+              "version_added": false,
+              "notes": "See <a href='https://webkit.org/b/78087'>bug 78087</a>."
             },
             "samsunginternet_android": {
-              "version_added": null
+              "version_added": "1.5"
```

Comment:
Is there a source for this? It doesn't quite square with the Blink version mapping (though it's my understanding that's not a perfect match for all features).

Reply:
No idea, my source was the existing data in https://github.com/mdn/browser-compat-data/blob/master/css/at-rules/media.json#L1314

This change is part of goal #1 listed above, "Harmonize the data for css.at-rules.media.resolution and css.types.resolution". My understanding is that the root support info for the resolution type and the support info for the resolution media query should match, since both are defined -- per the discussion in this PR -- as "at least one unit is supported with the resolution media query".

Reply:
Ah, I didn't see that the data was already in the other feature. Sorry about that. Leave this as is.